### DISCUSSION TITLE
pool instalation guide complete request

### DISCUSSION THREAD

**Discussion by astra23** [coinfoundry/miningcore#1585] [Q&A]
HI there anyone can someone help me to build mining pool for scrypt coins with complete guide without errors please becouse I already try something but cannot go on can someone help me about that pleeeeeeese

**Comment by oliverw** [comment]
Well, there is always https://store.miningcore.pro.

**Comment by btclinux** [comment]
I have a simple guide in Russian how to set up miningcore including known issues https://slabber.io/posts/498. I can translate it, but I think DeepL translator should be enough
